internal/cli/atlas/backup/exports/jobs/decribe.go

```diff
@@ -79,13 +79,15 @@ func DescribeBuilder() *cobra.Command {
 	}
 
 	cmd.Flags().StringVar(&opts.clusterName, flag.ClusterName, "", usage.ClusterName)
-	cmd.Flags().StringVar(&opts.bucketID, flag.BucketID, "", usage.BucketID)
+	cmd.Flags().StringVar(&opts.exportJobID, flag.ExportJobID, "", usage.ExportJobID)
+	cmd.Flags().StringVar(&opts.exportJobID, flag.BucketID, "", usage.ExportJobID)
 
 	_ = cmd.MarkFlagRequired(flag.ClusterName)
-	_ = cmd.MarkFlagRequired(flag.BucketID)
 
 	cmd.Flags().StringVar(&opts.ProjectID, flag.ProjectID, "", usage.ProjectID)
 	cmd.Flags().StringVarP(&opts.Output, flag.Output, flag.OutputShort, "", usage.FormatOut)
 
+	_ = cmd.Flags().MarkDeprecated(flag.BucketID, fmt.Sprintf("please use --%s instead", flag.ExportJobID))
```

Comment:
can you please mark the flags as mutually exclusive https://github.com/spf13/cobra/blob/main/user_guide.md#flag-groups

Reply:
then I think we are good to merge

Reply:
Thank you! I was looking for something like this